### ISSUE TITLE
Always on Top

### ISSUE THREAD

**Issue by Fruggy** [Snipaste/feedback#1754]
What happened to the Always on Top for a pinned image? I used to be able to turn it off but it isn't in the right click menu anymore.

**Comment by liulex** [comment]
It is only available for the PRO version. Something wrong with your activation status?

**Comment by Fruggy** [comment]
Yes, my store quit working right and I tried fixing it and now it's gone. I can't get it back without reinstalling the OS

**Comment by liulex** [comment]
Did you log in the Store app? Try purchasing from the tray menu again (don't worry, the store won't charge you twice if your account already bought it), then Snipaste PRO should be actived again.

**Comment by Fruggy** [comment]
My Microsoft Store wasn't working so I looked for a fix on Google. I found one where you basically uninstall it and then reinstall it but in my case it wouldn't reinstall so I have no store app on Windows anymore.

I right click the tray icon and the unlock PRO is greyed out so I assume it's the PRO version.

**Comment by liulex** [comment]
Nope, the greyed out version is the desktop version, which has no PRO version yet. You need to install the store version to enjoy premium features.

**Comment by Fruggy** [comment]
I wish I could but like I said my store app is gone now.

**Comment by liulex** [comment]
Then you need to get back the store app. I can do nothing on my side.

**Comment by deltaidea** [comment]
I think this issue is stale?
